plugins/modules/azure_rm_webappslot.py

```diff
@@ -16,7 +16,7 @@
 DOCUMENTATION = '''
 ---
 module: azure_rm_webappslot
-version_added: "2.8"
+version_added: "0.1.2"
```

Comment:
BTW: you cannot add new features in a patch release. This violates the principle of semantic versioning.